css/css-grid/parsing/display-invalid.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Layout Module Level 1: parsing display with invalid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#propdef-display">
+<meta name="assert" content="display supports the new values 'grid | inline-grid'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_invalid_value("display", "grid inline-grid");
+test_invalid_value("display", "none grid");
```

Comment:
Not a strong opinion but I don't see the need to have two separated tests valid vs invalid, instead of doing just one test checking both things.

That test could be name something like `grid-model/display-parsing-001.html` (the 001 suffix is always nice in case someone wants to add a new test about the same things in the future and won't need to do any rename).

Reply:
I find the separate files helpful when some browsers don't support a property.